api/src/main/java/org/apache/iceberg/actions/ActionsProvider.java

```diff
@@ -31,6 +31,11 @@ default SnapshotTable snapshotTable(String sourceTableIdent) {
 
   /** Instantiates an action to migrate an existing table to Iceberg. */
   default MigrateTable migrateTable(String tableIdent) {
+    return migrateTable(tableIdent, false);
+  }
+
+  /** Instantiates an action to migrate an existing table to Iceberg. */
+  default MigrateTable migrateTable(String tableIdent, Boolean dropBackup) {
```

Comment:
We should just keep this the same, and use the options already existing on MigrateTables.

Reply:
Done, Fixed in the latest commit.

Reply:
Better but not exactly what I meant, I think we still need an extra method on MigrateTableAction object (MigrateTableAction::dropBackup(boolean)).  Otherwise piggybacking with properties there would make it go to tableProperties?